src/Pos/TxHistory.hs

```diff
@@ -0,0 +1,241 @@
+{-# LANGUAGE CPP                  #-}
+{-# LANGUAGE InstanceSigs         #-}
+{-# LANGUAGE RankNTypes           #-}
+{-# LANGUAGE ScopedTypeVariables  #-}
+{-# LANGUAGE TemplateHaskell      #-}
```

Comment:
As for `InstanceSigs`, I don't actually remember what it does but stylish-haskell refused to process the file until I added it.